screenshot-testing/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply from: configs.androidLibrary
+
+dependencies {
+    implementation project(':stripe-ui-core')
+    implementation libs.compose.ui
+    implementation libs.compose.foundation
+    implementation libs.compose.material
+
+    // Implemented here instead of using plugin which implements with testImplementation
+    implementation("app.cash.paparazzi:paparazzi:${versions.paparazzi}") {
```

Comment:
I think this should actually be `api` because we expose `DeviceConfig` in the "public" API of the rule.

Reply:
Updated 👍